_posts/2015-11-23-my-first-couple-of-months-at-codurance.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: post
+name: my-first-couple-of-months-at-codurance
+title: 'My first couple of months at Codurance'
+date: 2015-11-23 09:00:00 +00:00
+author: Mani Sarkar
+image:
+    src: "/assets/img/custom/blog/Five\ Characteristics\ of\ a\ Great\ Company\ Culture.jpg"
+    attribution:
+        text: Together we are a formidable force
+tags:
+- craftsmanship
+- apprenticeship
+- community
+- mentor
+- mentee
+- mentoring
+---
+
+### Some background
+
+Some of you may know me from the various meetups in the city, especially my attendance at a number of [LJC](www.meetup.com/Londonjavacommunity/) and [LSCC](www.meetup.com/london-software-craftsmanship) meetup events. Attending these events I learnt about various conferences like **Devoxx**, **SoCraTes**, **JAX London**, **Java2Days**, **OpenFest**, and some others, and have had the good fortune of being at such events and even presenting at some of them. My subject was usually around [Adopt OpenJDK](https://java.net/projects/adoptopenjdk/pages/AdoptOpenJDK) whilst wearing my [LJC](www.meetup.com/Londonjavacommunity/) Advocate hat, but I also took the chance to present on other topics.
+
+During this time I met a lot of people with various levels of experience and my interest and urge to learn more about the Java/JVM platform, Code Quality, Software Design, XP Practices, Software Craftsmanship, etc..., were on the rise and saw no end. And whilst attending these events I came across [Sandro](https://twitter.com/sandromancuso) and [Mash](https://twitter.com/mashooq), who were in those days hosting [LSCC](www.meetup.com/london-software-craftsmanship) events. I went to many of [LSCC](www.meetup.com/london-software-craftsmanship) events, especially liked the hands-on sessions (which are still my favourite). These events always took place in the evenings and sometimes during weekends, and in most instances organised at [Skillsmatter](https://skillsmatter.com/).
+
```

Comment:
Remove the last line. "These events always... " It's irrelevant information and takes the reader's attention away from the blog.